src/containers/Home/Jobs/ListJob/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React,{useEffect,useState} from 'react'
+import Job from '../../../model/job';
+import { getJobs } from '../../../Redux/jobs/jobSlice';
+import {useSelector , useDispatch} from "react-redux"
+import { RootState } from '../../../Redux/store/store';
+export default function ListJob() {
+    const data = useSelector ((state:RootState) => state.jobs)
+    const dispatch = useDispatch();
+    useEffect(()=>{
+       dispatch(getJobs());
+    },[]);
+    const jobs= data.data;
+    const loading = data.loading;
+    if(loading){
+        return(
+            <div>Loading...</div>
+        )
+    }else{
+        return <>
+    {jobs?.map((item: Job, key: number) => {
+        return (      
+            <a key={key} >   
+                <li className={`list-group-item d-flex job`} data-testid="test100Jobs">
+                    <img className='job__img' alt='logo' src={item.img} />
```

Comment:
non-blocker, I see now you just have 10 imgs for first 10 jobs. `mockapi` support to generate the random img, you should try this (nice to have)

Reply:
i've tried it